tool/resources/org/antlr/v4/tool/templates/codegen/Kotlin/Kotlin.stg

```diff
@@ -959,7 +959,7 @@ public open class <lexer.name>(input: CharStream) : <superClass; null="Lexer">(i
     public object Modes {
         public const val DEFAULT_MODE: Int = 0
         <if(rest(lexer.modes))>
-        <rest(lexer.modes):{m | public const val <m>: Int = <i>}; separator="\n">
+        <rest(lexer.modes):{m | public const val MODE_<m; format="upper">: Int = <i>}; separator="\n">
```

Comment:
Just wondering why this is prefixed with `_MODE` now.

Reply:
Probably this is not necessary. If the name is uppercase it should not clash with keywords anyway

Reply:
@ftomassetti yup, that's the same logic we use for the other "namespaces".

Reply:
I think we should either:
* remove the `_MODE` suffix from `DEFAULT_MODE` and the `MODE_` prefix from the other modes
* keep `DEFAULT_MODE` and change `MODE_<m; format="upper">` into `<m; format="upper">_MODE`

What do you think @lppedd ? Any preference?

Reply:
@ftomassetti I'd say the most idiomatic is option 1

> remove the _MODE suffix from DEFAULT_MODE and the MODE_ prefix from the other modes

The suffix (or prefix) is redundant inside a `Modes` namespace imo.

You can keep `<m; format="upper">` tho, so we are sure they're always uppercased and no conflict will ever arise.

Reply:
Sounds good!